### ISSUE TITLE
a little question about the M and M_{^}

### ISSUE THREAD

**Issue by Raineasy** [kkkcx/DIFFender#3]
thanks for u advanced code !
i have a question about the first loss function in the paper ''employ cross-entropy loss by comparing estimated mask ˆ M with ground-truth mask M.'' where the ground-truth mask M could be found?
in the code i found the L1 is comparing the M_{^} with the label

I'd appreciate it if you could answer my questions.
